File: Java/UberX.java
package Java;

public class UberX extends Car{
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model){
        super(license, driver);
        this.brand = brand;
        this.model = model;
    }

    void printDataCar(){
        super.printDataCar();
        System.out.println("Brand: " + brand);
        System.out.println("Model: " + model);        
    }
    
}

File: PHP/uberX.php
<?php
require_once('car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }

    public function printDataCar() {
        parent::printDataCar();
        echo "
            Brand: $this->brand <br>
            Model: {$this->model} <br>
        ";
    }
}

?>